### PR TITLE
Adding some minor improvements to the scripts

### PR DESCRIPTION
**Pull request by malpiszon**
I'm abandoning my repo and I hope some improvements are useful to merge

### CODE REVIEW

File: .github/workflows/docker-publish.yml [hunk 1]
```diff
@@ -106,5 +106,6 @@ jobs:
           push: true
           tags: ${{ steps.meta.outputs.tags }}
           labels: ${{ steps.meta.outputs.labels }}
+          outputs: type=image,name=target,annotation-index.org.opencontainers.image.description=The NordVPN client for Docker
```
Comment (malpiszon) [inline .github/workflows/docker-publish.yml:109]: https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry#adding-a-description-to-multi-arch-images

Comment (tmknight) [reply .github/workflows/docker-publish.yml:109]: OK, this in place before change to a template approach.

File: nordvpn.dockerfile [hunk 2]
```diff
@@ -4,8 +4,7 @@ ARG UBUNTU_VER
 ARG NORDVPN_VERSION
 ARG TARGETARCH
 LABEL org.opencontainers.image.base.name="ubuntu:${UBUNTU_VER}"
-LABEL org.opencontainers.image.description DESCRIPTION
-LABEL org.opencontainers.image.licenses=GPL-3.0
+LABEL org.opencontainers.image.licenses=MIT
```
Comment (tmknight) [inline nordvpn.dockerfile:7]: Agreed

File: scripts/30-route
Comment (tmknight) [inline scripts/30-route:1]: Agreed for the sake of consistency

File: scripts/30-route6
Comment (tmknight) [inline scripts/30-route6:1]: Agreed; consistency and spelling

File: scripts/40-bypasslist
Comment (tmknight) [inline scripts/40-bypasslist:1]: OK, no harm

File: scripts/nord_login
Comment (tmknight) [inline scripts/nord_login:1]: Need to investigate this - I'm certain I left this out for a reason

**Review by tmknight — APPROVED** [review]
Approved for merge into develop

File: nordvpn.dockerfile [hunk 3]
```diff
@@ -4,8 +4,7 @@ ARG UBUNTU_VER
 ARG NORDVPN_VERSION
 ARG TARGETARCH
 LABEL org.opencontainers.image.base.name="ubuntu:${UBUNTU_VER}"
-LABEL org.opencontainers.image.description DESCRIPTION
```
[the remark this answers is not in the document]
Comment (malpiszon) [reply nordvpn.dockerfile:7]: Yes, but `=` was missing and `DESCRIPTION` wasn't defined (however I suppose the previous comment is the most important reason to move it, even it's not consistent right now).

File: scripts/nord_login
Comment (malpiszon) [reply scripts/nord_login:1]: I was wondering what could be the reason but I couldn't find a flow where the token is not provided and the script still displays `Login successful` (maybe this message needs to be moved?)

**Pull request merged**